src/2023.html

```diff
@@ -767,43 +767,773 @@ <h1 class="heading-1">THE CREATIVES</h1>
               feet beat
             </div>
           </div>
-          <div class="base-container w-container">
-            <div class="collection-list-wrapper w-dyn-list">
-              <div role="list" class="collection-list w-dyn-items">
-                <div role="listitem" class="collection-item w-dyn-item">
+          <div
+            data-w-id="12831666-34c3-9b9a-6260-37b0990e4207"
+            data-wf-id='["12831666-34c3-9b9a-6260-37b0990e4207"]'
+            class="base-container w-container"
+          >
+            <div
+              data-w-id="24586b92-2995-f2ef-5269-181fcb07fcb2"
+              data-wf-id='["24586b92-2995-f2ef-5269-181fcb07fcb2"]'
+              class="collection-list-wrapper w-dyn-list"
+            >
+              <div
+                data-w-id="24586b92-2995-f2ef-5269-181fcb07fcb3"
+                data-wf-id='["24586b92-2995-f2ef-5269-181fcb07fcb3"]'
+                role="list"
+                class="collection-list wf-grid w-dyn-items"
+              >
+                <div
+                  data-w-id="24586b92-2995-f2ef-5269-181fcb07fcb4_instance-0"
+                  data-wf-id='["24586b92-2995-f2ef-5269-181fcb07fcb4_instance-0"]'
+                  data-dyn-label="Collection Item 1"
+                  role="listitem"
+                  class="collection-item w-dyn-item"
+                >
+                  <img
+                    class="base-image creatives"
+                    src="https://uploads-ssl.webflow.com/62adfe360bb19444fa639741/64ff1696663c138a5f284199_Alexa.jpg"
```

Comment:
Hmm these aren't good... as the site will be deleted from there. I'll attempt to get the images and we can add them to this project.

Reply:
oh, okay! nice catch 👌 
I will download and add to the project